### PR TITLE
EZP-28270: As an Editor, I want to edit content from sub-items and MFU

### PR DESCRIPTION
**Pull request by dew326**
**Jira ticket:** https://jira.ez.no/browse/EZP-28270

### CODE REVIEW

File: src/modules/multi-file-upload/components/upload-list/upload.item.component.js [hunk 1]
```diff
@@ -319,7 +309,23 @@ export default class UploadItemComponent extends Component {
...
+     * @method createEditLink
+     * @memberof UploadItemComponent
+     */
+    createEditLink() {
+        if (!this.state.struct) {
+            return;
+        }
+
+        return Routing.generate('ez_content_draft_create', {
+            'contentId': this.state.struct.Content._id,
```
Comment (sunpietro) [inline src/modules/multi-file-upload/components/upload-list/upload.item.component.js]: Isn't just `contentId:` enough?

File: src/modules/sub-items/components/table-view/table.view.item.component.js [hunk 2]
```diff
@@ -86,6 +86,18 @@ export default class TableViewItemComponent extends Component {
...
 
+    /**
+     * Creates edit link
+     *
+     * @method createEditLink
+     * @memberof TableViewItemComponent
+     */
+    createEditLink() {
+        return Routing.generate('ez_content_draft_create', {
+            'contentId': this.props.content._id,
```
Comment (sunpietro) [inline src/modules/sub-items/components/table-view/table.view.item.component.js]: Same as above

Comment (sunpietro) [reply src/modules/sub-items/components/table-view/table.view.item.component.js]: Missing return doc

File: src/modules/multi-file-upload/components/upload-list/upload.item.component.js [hunk 3]
```diff
@@ -319,7 +309,23 @@ export default class UploadItemComponent extends Component {
         return FILE;
     }
 
-        /**
+    /**
+     * Creates edit link
+     *
+     * @method createEditLink
+     * @memberof UploadItemComponent
+     */
```
Comment (sunpietro) [inline src/modules/multi-file-upload/components/upload-list/upload.item.component.js]: Missing return doc

File: src/modules/sub-items/components/table-view/table.view.item.component.js [hunk 4]
```diff
@@ -145,12 +157,12 @@ export default class TableViewItemComponent extends Component {
                 <td className="c-table-view-item__cell--translations">{content.mainLanguageCode}</td>
                 <td className="c-table-view-item__cell--actions">
                     <div>
-                    <button className="c-table-view-item__btn--edit">
+                    <a href={this.createEditLink()} className="c-table-view-item__btn--edit">
```
Comment (sunpietro) [inline src/modules/sub-items/components/table-view/table.view.item.component.js:161]: Is underlining turned off for links?

**Review by sunpietro — CHANGES_REQUESTED** [review]

**Comment by dew326** [comment]
@sunpietro fixed

**Review by sunpietro — APPROVED** [review]

**Pull request merged**